Exiled.Events/Patches/Events/Warhead/Detonated.cs

```diff
@@ -41,8 +79,8 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                     new CodeInstruction(OpCodes.Call, Method(typeof(Warhead), nameof(Warhead.OnDetonated))),
                 });
 
-            for (int z = 0; z < newInstructions.Count; z++)
-                yield return newInstructions[z];
+            foreach (var instruction in newInstructions)
+                yield return instruction;
```

Comment:
Using `for` with `z` iterator instead of `foreach` is a exiled tradition, dont break it🙃